src/engine/node/node.render.ts

```diff
@@ -248,7 +250,7 @@ export function updateLocalNodeResources(
     updateNodeCamera(ctx, activeSceneResource.scene, node, nodeView);
     updateNodeLight(ctx, activeSceneResource.scene, node, nodeView);
     updateNodeReflectionProbe(ctx, activeSceneResource.scene, node, nodeView);
-    updateNodeMesh(ctx, activeSceneResource, node, nodeView);
+    updateNodeMesh(ctx, activeSceneResource, node, nodeView, gameDeltaTime);
```

Comment:
Do we only need to use gameDeltaTime for meshes? Or would the same need to be used for cameras, lights, etc?

Can we pull this off the context inside `updateTransformFromNode` instead of passing it here?